ts.override.jdk17.properties

```
@@ -53,10 +53,11 @@ command.testExecuteEjbEmbed=com.sun.ts.lib.harness.ExecTSTestCmd \
         -Djava.util.logging.config.file=${javaee.home}/domains/domain1/config/logging.properties \
         -DAS_DERBY_INSTALL=${javaee.home}/../javadb \
         -Dtest.ejb.stateful.timeout.wait.seconds=${test.ejb.stateful.timeout.wait.seconds} \
-        -Ddeliverable.class=${deliverable.class} $testExecuteClass $testExecuteArgs
+        -Ddeliverable.class=${deliverable.class} \
+        --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED $testExecuteClass $testExecuteArgs
 
 # disable variable substitution in TCK tests, set UTC timezone
-vi.jvm.options=-Doracle.jdbc.J2EE13Compliant=true:-Xmx4095m:-Dj2eelogin.name=${user}:-Dj2eelogin.password=${password}:-Deislogin.name=${user1}:-Deislogin.password=${password1}:-Dtest.ejb.stateful.timeout.wait.seconds=${test.ejb.stateful.timeout.wait.seconds}:-DwebServerPort.2=${webServerPort.2}:-DwebServerHost.2=${webServerHost.2}:-Dcsiv2.save.log.file=${harness.log.traceflag}:-Djavax.xml.accessExternalStylesheet=all:-Djavax.xml.accessExternalDTD=file,http:-Dfish.payara.substitution.disable=true:-Duser.timezone=UTC:--add-opens=java.base/jdk.internal.vm.annotation=ALL-UNNAMED
+vi.jvm.options=-Doracle.jdbc.J2EE13Compliant=true:-Xmx4095m:-Dj2eelogin.name=${user}:-Dj2eelogin.password=${password}:-Deislogin.name=${user1}:-Deislogin.password=${password1}:-Dtest.ejb.stateful.timeout.wait.seconds=${test.ejb.stateful.timeout.wait.seconds}:-DwebServerPort.2=${webServerPort.2}:-DwebServerHost.2=${webServerHost.2}:-Dcsiv2.save.log.file=${harness.log.traceflag}:-Djavax.xml.accessExternalStylesheet=all:-Djavax.xml.accessExternalDTD=file,http:-Dfish.payara.substitution.disable=true:-Duser.timezone=UTC:--add-opens=java.base/jdk.internal.vm.annotation=ALL-UNNAMED:--add-opens=java.base/java.lang=ALL-UNNAMED:--add-opens=java.base/java.util=ALL-UNNAMED:--add-opens=java.base/java.io=ALL-UNNAMED: --add-opens=java.base/sun.net.www.protocol.jar=ALL-UNNAMED:--add-opens=java.naming/javax.naming.spi=ALL-UNNAMED
```

Errant space? _Probably_ harmless
```suggestion
vi.jvm.options=-Doracle.jdbc.J2EE13Compliant=true:-Xmx4095m:-Dj2eelogin.name=${user}:-Dj2eelogin.password=${password}:-Deislogin.name=${user1}:-Deislogin.password=${password1}:-Dtest.ejb.stateful.timeout.wait.seconds=${test.ejb.stateful.timeout.wait.seconds}:-DwebServerPort.2=${webServerPort.2}:-DwebServerHost.2=${webServerHost.2}:-Dcsiv2.save.log.file=${harness.log.traceflag}:-Djavax.xml.accessExternalStylesheet=all:-Djavax.xml.accessExternalDTD=file,http:-Dfish.payara.substitution.disable=true:-Duser.timezone=UTC:--add-opens=java.base/jdk.internal.vm.annotation=ALL-UNNAMED:--add-opens=java.base/java.lang=ALL-UNNAMED:--add-opens=java.base/java.util=ALL-UNNAMED:--add-opens=java.base/java.io=ALL-UNNAMED:--add-opens=java.base/sun.net.www.protocol.jar=ALL-UNNAMED:--add-opens=java.naming/javax.naming.spi=ALL-UNNAMED
```

---

Hmm, are these new options actually doing anything? Have you tested without them there?
These `--add-opens` should be defined by default in the domain.xml

---

@Pandrex247 I think so. I incrementally added these based on log errors saying missing "opens". Presumably these are for the JVM running the test instead of the server itself?

---

I think this particular option is used for adding/removing JVM options from the server, not in appclient or anything (that's the other options you've touched).

The default domain.xml Should™ have these anyway: https://github.com/payara/Payara/blob/main/appserver/admin/gf_template/src/main/resources/config/domain.xml#L200-L226

---

Thanks @Pandrex247 . I've pushed a commit and will re-run CI to verify